### ISSUE TITLE
Consider explaining that code interpreter sessions can time out

### ISSUE THREAD

**Issue by earth2marsh** [allisonmorrell/gptbuilder#13]
I was using GPT Builder yesterday, but I stepped away during the process with the assumption that I could pick it up later. What I didn't realize is that the GPT depends on the code interpreter, and those sessions can time out! 

If the GPT could explain that and set expectations at the beginning of how long the process might take and that it can time out, that might have helped me. (Screenshot for context. Also, thanks for publishing this on GitHub!)

![Screenshot 2023-11-19 10 54 44 AM](https://github.com/allisonmorrell/gptbuilder/assets/54582/fe20303a-7ec1-4e18-a57b-b769a7f8ff21)

**Comment by allisonmorrell** [comment]
Yes, I should add more explicit instructions to download files. 

If you need to recover the session, you can usually tell the bot that the code execution state is reset and get it to rewrite them, or at worst open the code and copy-paste. But it's not user-friendly.

I'm also unsure whether the current very long process is ideal - it helps the result by forcing chain of thought, but it's also time-consuming.
